src/Helpers/PjsConsole.js

```diff
@@ -117,19 +122,11 @@ module.exports = function PjsConsole(document) {
   };
 
   e.println = function () {
-    if(!added) {
-      document.body.appendChild(style);
-      document.body.appendChild(e.wrapper);
-      added = true;
-    }
     var args = Array.prototype.slice.call(arguments);
     args.push('<br>');
     e.print.apply(e, args);
-    if (e.BufferArray.length > e.BufferMax) {
-      e.BufferArray.splice(0, 1);
-    } else {
-      e.javaconsole.scrollTop = e.javaconsole.scrollHeight;
-    }
+    if (e.BufferArray.length > e.BufferMax) e.BufferArray.splice(0, 1);
```

Comment:
please don't include formatting changes in a functional PR, especially in this case: curly brackets serve a purpose, don't slow down code, and make things easier to read. Removing them is not a good idea =)